Cargo.toml

```diff
@@ -287,3 +289,6 @@ github-custom-job-permissions = { "build-docker" = { packages = "write", content
 install-updater = false
 # Path that installers should place binaries in
 install-path = "CARGO_HOME"
+
+[workspace.metadata.cargo-shear]
+ignored = ["red_knot_test"]
```

Comment:
It's ignored because nothing depends on it and it's not a binary, so `cargo shear` thinks that it is unused in the root workspace.

I think the need for this ignore will be eliminated if we move the actual tests to a different red-knot crate, as you suggested elsewhere. Because then that crate will depend on `red_knot_test`.